streampark-console/streampark-console-service/src/assembly/script/data/mysql-data.sql

```diff
@@ -110,6 +110,7 @@ insert into `t_menu` values (100053, 100001, 'types', null, null, 'user:types',
 -- Records of t_role
 -- ----------------------------
 insert into `t_role` values (100001, 'developer', 'developer', now(), now(), null);
+insert into `t_role` values (100002, 'team_admin', 'Team Admin has all permissions inside the team.', now(), now(), null);
```

Comment:
how about team_admin change to team_leader?

Reply:
Hi @wolfboys , thanks for your review, it's a good idea.

I have a little concern. To facilitate managing jobs, a team may have multiple `team_admin`, including some developers.

`team_leader` represents the leader of the team. There may be some ambiguity here.